src/lib.rs

```diff
@@ -135,6 +129,11 @@ impl AtomicF64 {
             .swap(u64::from_ne_bytes(v.to_ne_bytes()), Ordering::SeqCst);
         f64::from_ne_bytes(prev.to_ne_bytes())
     }
+
+    pub fn fetch_add(&self, v: f64) -> f64 {
```

Comment:
This one is a bit dangerous. It's a method that suggests to be thread safe but actually isn't!
In practise this is not an issue because only the render thread (a single thread) will call methods that update the value. The control thread (possibly multi-threaded) only reads the value, which is then safe.
In any case I suggest we start using  https://crates.io/crates/atomic_float